tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -748,6 +748,20 @@ void SpirvEmitter::doDecl(const Decl *decl) {
   }
 }
 
+RichDebugInfo *
+SpirvEmitter::getOrCreateRichDebugInfo(const SourceLocation &loc) {
```

Comment:
oh, i had commented on creating this function in an earlier review. heh. It's already done